tests/kubernetes/test_pod_generator.py

```diff
@@ -658,11 +658,12 @@ def test_deserialize_model_file(self):
     def test_pod_name_confirm_to_max_length(self, _, pod_id):
         name = PodGenerator.make_unique_pod_id(pod_id)
         assert len(name) <= 253
-        parts = name.split(".")
-        if len(pod_id) <= 63:
+        parts = name.split("-")
+        if len(pod_id) <= 63 - 33:
             assert len(parts[0]) == len(pod_id)
         else:
             assert len(parts[0]) <= 63
+        assert len(parts[1]) == 32
         assert len(parts[1]) <= 63
```

Comment:
```suggestion
```

Don't need to check it's <=63 when we've just asserted it's 32.

Reply:
good catch